src/crudeditor-lib/views/search/selectors.js

```diff
@@ -71,11 +71,19 @@ export const
 
   getViewModelData = wrapper((storeState, {
     model: modelMeta,
-    ui: uiMeta,
+    ui: {
+      Spinner,
+      search: {
+        standardOperations: standardOpsConfig,
```

Comment:
Naming should follow **model.ui** convention since we change nothing in the **standardOperations** property behaviour.  Better naming?  Then  **standardOperations** property on **model.ui** should be changed as well

Reply:
Fixed